.github/workflows/ci.yaml

```diff
@@ -49,6 +49,8 @@ jobs:
 
     - name: Run integration tests
       run: tox -vve integration -- --model testing
+      env:
+        KUBECONFIG: /home/runner/.kube/config
```

Comment:
Is this necessary because the tests themselves use kubectl/lightkube?

Reply:
Also, I think by default kubectl looks in `~/.kube/config`, is that different than this?